src/System.IO.IsolatedStorage/tests/System/IO/IsolatedStorage/IsoStorageTest.cs

```diff
@@ -27,5 +28,62 @@ public static IEnumerable<object[]> ValidScopes
                 };
             }
         }
+
+        public enum PresetScopes
+        {
+            UserStoreForApplication,
+            UserStoreForAssembly,
+            UserStoreForDomain,
+            // MachineStoreForAssembly,
+            // MachineStoreForApplication,
+            // MachineStoreForDomain
```

Comment:
Need to implement the ACLing. I've put the comment most everywhere- missed this one. I'll be adding that this week and uncommenting these.